Introduce Noetherian (P208) (#999)
pi-base/data: theorems/T000251.md

if:
  P000129: true
then:
  P000208: true
---

> The ascending chain condition holds since there is only one nonempty open set.

Reviewer: ```suggestion
The ascending chain condition on open sets holds since there are at most two open sets.
```
